python/orca/src/bigdl/orca/automl/model/base_keras_model.py

```diff
@@ -224,13 +231,23 @@ def _get_required_parameters(self):
     def _get_optional_parameters(self):
         return {"batch_size"}
 
+    def _get_input_shape(self):
+        past_seq_len = self.config['past_seq_len']
+        input_feature_num = self.config['input_feature_num']
+        batch_size = self.config.get('batch_size', 32)
+        input_shape = tf.random.normal((batch_size, past_seq_len, input_feature_num))
+        return input_shape
 
 class KerasModelBuilder(ModelBuilder):
 
-    def __init__(self, model_creator):
+    def __init__(self, model_creator, optimizer, loss):
```

Comment:
why we need this `optimizer` and `loss`. I think there should be a good reason since may code change is related to this

Reply:
`Modulebuilder` will init a model, if optimizer and loss are not specified, then the default parameters in `model_creator` will be used.

Reply:
optimizer and loss can be moved to search_space in automodels